CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+
+- Compatibility with Wagtail 4.2 - [#673](https://github.com/wagtail/wagtail-localize/pull/673) @katdom13
+
+### Removed
+
+- Dropped tests with wagtail versions 2.16 to 4.0 from the workflow - [#673](https://github.com/wagtail/wagtail-localize/pull/673) @katdom13
```

Comment:
Please use `Wagtail` everywhere

Reply:
the other bit is.. move both under `### Changed` since changes things. e.g. there is no compatibility fix other than adding it to the test matrix.

Reply:
Hi @zerolab ,
Please see the changes to `CHANGELOG.md`

- Updated `wagtail` to `Wagtail`
- Moved new change logs to `### Changed`
- Combine the new change logs since they have the same PR and are related.

https://github.com/wagtail/wagtail-localize/pull/673/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR10-R13

Thanks!